skt/reporter.py

```diff
@@ -328,9 +330,9 @@ def getbuildfailure(self):
         return result
 
     def getjobresults(self):
-        result = ["\n-----------------------"]
+        result = ['\n' + '=' * 70 + '\n']
```

Comment:
I like how Perl has a separate operator for this. Operator overloading should be used carefully.